services/core-web/src/components/Forms/PermitAmendmentForm.js

```diff
@@ -16,6 +16,13 @@ import PermitAmendmentFileUpload from "@/components/mine/Permit/PermitAmendmentF
 const originalPermit = "OGP";
 const amalgamtedPermit = "ALG";
 
+const ROLES_ALLOWED_TO_CREATE_HISTORICAL_AMENDMENTS = [
```

Comment:
Can you move this to the permissions.js file to keep all roles in one spot?

Reply:
Also, should this be handled through a group on keycloak? @pdittaro thoughts?

Reply:
Correct, we should never use the 'mds' prefixed roles within the application, they are just for management in Keycloak. This should be a new permission if that's the case, i.e. prefixed with "core_"